src/libponyc/type/cap.h

```diff
@@ -7,6 +7,12 @@
 
 PONY_EXTERN_C_BEGIN
 
+/**
+ * Normalizes the capability and alias modifier to apply aliasing
+ * and remove unnecessary ephemeral modifiers.
+ */
+void cap_aliasing(token_id* cap, token_id* eph);
```

Comment:
Is this worth leaving in? Or should we revert

Reply:
It is still used elsewhere.